### PR TITLE
UI art connection

### PR DESCRIPTION
**Pull request by Maxi-F**

### CODE REVIEW

**Comment by Maxi-F** [comment]
[[Implementation] Settings UI](https://trello.com/c/qjrd63Jk/312-implementation-settings-ui)

**Comment by Maxi-F** [comment]
[[Implementation] New Visor UI](https://trello.com/c/yp7whXaO/352-implementation-new-visor-ui)

**Comment by Maxi-F** [comment]
[[Implementation] Velocimeter of Gameplay UI](https://trello.com/c/nGXDuYVW/313-implementation-velocimeter-of-gameplay-ui)

File: Assets/Scripts/UI/Bars/HealthBar.cs [hunk 1]
```diff
@@ -53,19 +53,18 @@ public void HandleReset(int currentHp)
...
         public void HandleInit(int maxValue)
         {
-            Debug.Log($"Handle init {maxValue}");
             slider.maxValue = maxValue;
             slider.value = maxValue;
         }
         
         public void HandleTakeDamage(int currentHealth)
         {
+            Debug.Log($"here? {currentHealth}");
```
Comment (NicolasLaure) [inline Assets/Scripts/UI/Bars/HealthBar.cs:62]: "Here? {currentHealth}"?

**Review by NicolasLaure — APPROVED** [review]

**Pull request merged**